public/index.html

```diff
@@ -24,7 +21,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Gnosis Safe App</title>
+    <title>Gnosis Easy Auction</title>
```

Comment:
We want to call it Gnosis Auction, although in the Smart contracts it says easy auction :)